content/projects/java-specific/hello-sql-and-java/_index.md

```diff
@@ -1,3 +1,54 @@
 ---
-title: SQL + Java Helloworld
+title: SQL Extended
 ---
+
+We have had many sessions about JDBC Templates, JPA and Hibernate, so I would hope that you now have a clue of what we are talking about.
```

Comment:
This sentence is only relevent ofr one group of people, if we use this content for the C42 JAva devs then this sentence might not make any sense